pkg/installer/upgrade.go

```diff
@@ -55,7 +55,7 @@ func Upgrade(uninstallConfig *apiv1.KubectlStorageOSConfig, installConfig *apiv1
 	time.Sleep(30 * time.Second)
 
 	// install new storageos operator and cluster
-	err = installer.installStorageOS(installConfig)
+	err = installer.Install(installConfig)
```

Comment:
I think better to not be clever here and call the installer as it is. If include etcd is false - as it is - it will skip installation of it so will do the same :D